### PR TITLE
Add info on azure LB

### PR DESCRIPTION
**Pull request by CordeliaLopez**
### What does this PR do?

This PR makes clients aware that they need to have the azure integration installed in order to see Azure LB data.

### Motivation

The added functionality to NPM

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/network_monitoring/performance/setup.md [hunk 1]
```diff
@@ -132,6 +132,8 @@ To enable network performance monitoring with the Datadog Agent, use the followi
 
     **Note**: If the `systemctl` command is not available on your system, run the following command instead: `sudo service datadog-agent restart`
 
+6. [Optional] Make sure the Azure integration is installed for more visibility into Azure load balancers. This feature is currently not available in the Govcloud region.
```
Comment (buraizu) [inline content/en/network_monitoring/performance/setup.md:135]:
```suggestion
6. Optionally, install the Azure integration for more visibility into Azure load balancers.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, just made a small update per our contributing guidelines. Would you be open to using a site-region selector to display the information for only non-Govcloud users?

**Comment by CordeliaLopez** [comment]
> Thanks, just made a small update per our contributing guidelines. Would you be open to using a site-region selector to display the information for only non-Govcloud users?

@buraizu sure, do you have any documentation you can point me to in order to see how that's used?

**Review by buraizu — CHANGES_REQUESTED** [review]
@CordeliaLopez thanks for confirming! I've added in the `site-region` shortcode and confirmed that the content is not visible for users with the `us-1 fed` documentation site selected. I'll follow up internally with a link to further details.

**Review by buraizu — APPROVED** [review]

**Pull request merged**